### ISSUE TITLE
Clean Up README

### ISSUE THREAD

**Issue by julietanarchy** [anarchy-ai/LLM-VM#261]
something more barebones, cleaner, easier to understand, made for github local application rather than browser application

**Comment by jahlela** [comment]
Community Vibes: add link to discord https://discord.com/channels/1075227138766147656/1147542772824408074

**Comment by bilal-aamer** [comment]
@jahlela Isnt the discord link already there currently in the ReadMe.

**Comment by jahlela** [comment]
@bilal-aamer That's true, but this is a nice place we can add the link directly to the call to action, so they can click it right when they're thinking about it

**Comment by bilal-aamer** [comment]
Got you @jahlela, for the virtual environment, it would be better to give snippets towards two of the most used methods i.e. `venv` and `conda`

**Comment by SrijanSahaySrivastava** [comment]
Hello, may i work on this. I can utilize the help of GPT and my own knowledge to draft a optimized README.

**Comment by lucylililiwang** [comment]
Hi! Can I please work on this question too? Thank you!
